### PR TITLE
Classwork 3 - Reddit

### PR DESCRIPTION
**Pull request by klikithareddy**
CREATING REDDIT APP ACCOUNT AND SAMPLE APPLICATION:

Create a Reddit developer account and create a sample app.

AUTHENTICATION AND IMPORTING PRAW MODULE :

Here , we can give our user id and secret key which is generated in Reddit application for authentication.
PRAW, or the Python Reddit API Wrapper, is a Python package that provides a convenient way to interact with the Reddit API. The Reddit API allows developers to access and manipulate data from Reddit, such as retrieving posts, comments, user information, and more

CHOOSE TRENDING TOPIC FROM REDDIT:

We have selected "bitcoin" subreddit using PRAW.
headlines = set() initializes an empty set called headlines to store unique titles of posts.
Print statement various information about each post, such as its title, ID, author, creation time, score, upvote ratio, and the URL of the post.

In summary, the code fetches information about posts in the "bitcoin" subreddit, prints details about each post, and collects unique post titles in the headlines set. The total number of unique headlines is then printed at the end.

### CODE REVIEW

**Review by nikshepkulli — COMMENTED** [review]
missing header blocks!
